runelite-client/src/main/java/net/runelite/client/plugins/poh/PohPlugin.java

```diff
@@ -217,6 +228,37 @@ else if (actorName != null)
 
 	}
 
+	@Subscribe
+	public void onGameTick(GameTick gameTick)
+	{
+		if (!config.notifyBurner() || incenseBurners.size() == 0)
+			return;
+
+		getIncenseBurners().forEach((tile, burner) ->
+		{
+			if (BURNER_LIT.contains(burner.getId()) && burner.checkIfShouldNotify())
+			{
+				if (!recentlySentNotification)
```

Comment:
This can be moved in the upper if statement

Reply:
Its in its own if statement because if burner.checkIfShouldNotify() returns true, then the burner class will also have marked itself as notified on its internal variable and won't return true going forwards.

As such making sure burner.checkIfShouldNotify() fires before I suppress the notification means it won't just wait until after the suppression ends and then fire again.